packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -500,11 +570,18 @@ class MultiFrameImageStreamCompleter extends ImageStreamCompleter {
   ///
   /// [codec] is a future for an initialized [ui.Codec] that will be used to
   /// decode the image.
+  ///
   /// [scale] is the linear scale factor for drawing this frames of this image
   /// at their intended size.
+  ///
+  /// [chunkEvents] is an optional stream of notifications about the loading
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar
```suggestion
  /// The [chunkEvents] parameter is an optional stream of notifications about the loading
```